model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -846,32 +868,45 @@ async def create_vllm_bundle(
             final_weights_folder,
         )
 
-        vllm_cmd = f"python -m vllm_server --model {final_weights_folder} --tensor-parallel-size {num_shards} --port 5005"
+        vllm_cmd = ""
+
+        if multinode and is_leader:
+            vllm_cmd += "/workspace/init_ray.sh leader --ray_cluster_size=$RAY_CLUSTER_SIZE --own_address=$K8S_OWN_POD_NAME.$K8S_LWS_NAME.$K8S_OWN_NAMESPACE.svc.cluster.local; "
```

Comment:
is there a reason this ends in a `; ` but the worker one doesn't?

Reply:
oh i guess you don't start the vllm server on the worker node. Can we actually make this a separate subcmd (e.g. assign it to `ray_cmd` or something like that and rely on line 912 to add the `;`.